typings/index.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="globals/selenium-webdriver/index.d.ts" />
```

Comment:
Added a readme file. We will remove this folder when we can install these typings with `npm install @types/selenium-webdriver@2.53.1`.